graphene_django_extras/types.py

```diff
@@ -311,7 +311,7 @@ class Meta:
     def __init_subclass_with_meta__(cls, serializer_class=None, queryset=None, only_fields=(), exclude_fields=(),
                                     pagination=None, input_field_name=None, output_field_name=None,
                                     results_field_name=None, nested_fields=False, filter_fields=None, description='',
-                                    filterset_class=None, **options):
+                                    filterset_class=None, interfaces=(), **options):

```

Comment:
There is a type mismatch by using () here. An empty dictionary item will return None whereas this takes a default of a tuple. No idea if this matters but it's a thing

Reply:
It should be fine as it calls super for `__init_subclass_w`ith_meta__`  and in graphene.ObjectType https://github.com/graphql-python/graphene/blob/master/graphene/types/objecttype.py#L28 it set default for `interfaces` to a tuple in the arguments. This is doing the same one step earlier.

Reply:
Yes, but you pass along `interfaces=kwargs.get('interfaces')`, which if not provided by the caller translates to `interfaces=None`

Reply:
Are you talking about this scenario?
```
class A(object):
   @classmethod
   def test(cls, interfaces=(), **kwargs):
     return (type(interfaces), interfaces)

class B(A):
   @classmethod
   def test(cls, interfaces=(), **kwargs):
     return super(B, cls).test(interfaces=interfaces, **kwargs)
 
B.test(**{"test": True, "interfaces": None})
(<type 'NoneType'>, None)
```

I don' t think this is an issue as kwargs come from Meta class in the ModelType, and should only be assigned to a tuple (or left out if no interface is implemented).

Reply:
I'm talking about this scenario:

```
def foo(a=()):
  for x in a:
    print('do something with', a)

some_dict = {}
foo(a=some_dict.get('a'))
```
Where type type mismatch of a missing dictionary element causes a traceback